.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
Should this have a `.ruby-version` given that it's a Gem and used as a library? What was the reason for adding it?